lib/box-client.js

```diff
@@ -386,7 +386,12 @@ BoxClient.prototype.setIPs = function(ips) {
 	var validIPs = ips.filter(function(ipString) {
 		return isIP(ipString);
 	});
-	this.setCustomHeader(HEADER_XFF, validIPs.join(', '));
+	validIPs = validIPs.join(', ');
```

Comment:
validIPs are used twice after join(). It is not working for the second statement.